helm_deploy/values-prod.yaml

```diff
@@ -83,7 +83,6 @@ redis:
 metabase:
   priorAuthorityDashboardIds: ''
   nsmDashboardIds: ''
-  priorAuthorityAutograntDashboardIds: ''
```

Comment:
Are you sure it's safe to remove this? `deployment.yaml` still references it when setting `METABASE_PA_AUTOGRANT_DASHBOARD_IDS` (which is admittedly unused). I think helm might break if a referenced key doesn't exist in a values file.

Reply:
good shout - will remove from helm too

Reply:
Cool, then for completeness it would be worth removing `METABASE_PA_AUTOGRANT_DASHBOARD_IDS` from dashboard_controller_spec, and `priorAuthorityAutograntDashboardIds` from values-dev.yaml

Reply:
updated